### ISSUE TITLE
Accessibility issues with tabbing and focus

### ISSUE THREAD

**Issue by danieliser** [shahabyazdi/react-multi-date-picker#245]
Great package, but we found a few accessibility issues that should likely be addressed in the package rather than on our end.

1. No default focus state for calendar dates. The demo has an outline but that is the default browser outline, there should be an explicit one by default given your controlling tab order & states.
2. The month/year clickable items in the header are currently spans and cannot be tabbed to. Either convert them to buttons or give them a tabindex.

**Comment by shahabyazdi** [comment]
I'm working on it!

**Comment by danieliser** [comment]
@shahabyazdi - Awesome, glad to hear it.

**Comment by shahabyazdi** [comment]
Fixed in v4.5.0.
